lightning/src/ln/channelmanager.rs

```diff
@@ -2249,7 +2250,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					break None;
 				}
 				{
-					let mut res = VecWriter(Vec::with_capacity(chan_update.serialized_length() + 8 + 2));
+					let mut res = VecWriter(Vec::with_capacity(chan_update.serialized_length() + 2 + 8 + 2));
```

Comment:
I would like to have a very tiny comment that documents what this sum will do!

This make it harder to read for people that switch implementation during the day

Reply:
> I would like to have a very tiny comment that documents what this sum will do!
I'm not sure that a comment would provide much more information that is not self-explanatory? Maybe `chan_update.serialized_length() + mem::size_of<u16>() + mem::size_of<u64> + mem::size_of<u16>()` would be more expressive, but would also render the line unnecessarily long? Also, this only reserves a certain amount of vector capacity, nothing would really break if the calculation would be off by a byte or two.

> This make it harder to read for people that switch implementation during the day
Mh, how does this relate to other implementations?

Reply:
Mh? I mean a comment to explain the meaning of the number  `// to the serialization length we sum up: 2 (flag) + 8 (whatever) + 2(whatever) `

BTW, it is not relevent!